src/ext/draft/check_for_update.rs

```diff
@@ -20,9 +20,7 @@ unsafe impl Sync for clap_check_for_update_info {}
 #[repr(C)]
 #[derive(Copy, Clone)]
 pub struct clap_plugin_check_for_update {
-    // FIXME: This `clap_host` should probably be a `clap_plugin`, see
-    //        https://github.com/free-audio/clap/issues/79
-    pub check: unsafe extern "C" fn(plugin: *const clap_host, include_beta: bool),
+    pub check: unsafe extern "C" fn(plugin: *const clap_plugin, include_beta: bool),
```

Comment:
Should be `include_preview`: https://github.com/free-audio/clap/blob/469541c05a0fd59f123422dda3d792ee0f73bc5f/include/clap/ext/draft/check-for-update.h#L21

Reply:
Fixed.